src/extensions/wcf/Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater/Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
I switched from 2.0 to 2.1 because 2.1 has the overload option Contains(string, StringComparison), which could save a little bit time compared to Contains(string). But I think the flexibility might be more preferable in this case?

@mconnew Matt, What's your opinion on this? :)

Reply:
Yea, the new `String.Contains` overload in 2.1 is nice. But you can get the same behavior on netstandard2.0 by using `string.IndexOf(string, StringComparison)` and just checking that the result is >= 0.

Reply:
Isn't the only difference is supported frameworks between netstandard2.0 and netstandard2.1 whether you can run on .NET Framework or not? Unless I've missed something, every other in support framework can use 2.1

Reply:
Yes, but it's possible that in the future we may want to re-use some of this code in a VS extension to enable upgrade scenarios to work there and we'd need NetFx if we wanted to run in-proc.

Reply:
Hi Mike! Thanks for providing those contexts and that makes sense. I will change it back to .NET2.0.